### PR TITLE
Simple JupyterLab-Git and GitHub support

### PR DESCRIPTION
**Pull request by oranheim**

### CODE REVIEW

File: docker/jupyter/Dockerfile [hunk 1]
```diff
@@ -49,6 +49,26 @@ RUN python3 -m pip install toree && \
...
+RUN echo "**** install git-cli ****" && \
+    apk add --no-cache bash git openssh-client
+
+RUN echo "**** build jupyterlab-git ****" && \
+    apk add --no-cache yarn && \
+    git clone "https://github.com/jupyterlab/jupyterlab-git" && \
+    cd jupyterlab-git && \
+    sed -i 's/jlpm/yarn/g' package.json && \
+    yarn build:labextension && \
+    jupyter labextension install jupyterlab_git/labextension/jupyterlab-git-0.20.0-rc.0.tgz
```
Comment (takvamborgen) [inline docker/jupyter/Dockerfile:61]: Is it not easier to just install it directly with `pip install jupyterlab-git==0.20.0-rc.0`?

Comment (oranheim) [reply docker/jupyter/Dockerfile:61]:
```
**** build jupyterlab-git ****
ERROR: Could not find a version that satisfies the requirement jupyterlab-git==0.20.0-rc.0? (from versions: 0.1.1, 0.1.2, 0.2.0, 0.2.2, 0.3.0, 0.4.4, 0.5.0, 0.6.0, 0.6.1, 0.8.0, 0.8.1, 0.9.0rc1, 0.9.0, 0.9.1, 0.10.0, 0.10.1rc0, 0.10.1, 0.20.0rc0)
ERROR: No matching distribution found for jupyterlab-git==0.20.0-rc.0?
```

Comment (takvamborgen) [reply docker/jupyter/Dockerfile:61]: `Successfully installed GitPython-3.1.1 colorama-0.4.3 gitdb-4.0.4 jupyterlab-git-0.20.0rc0 nbdime-2.0.0 smmap-3.0.2`

Comment (oranheim) [reply docker/jupyter/Dockerfile:61]: Works now. I had a typo in version number.

Comment (oranheim) [reply docker/jupyter/Dockerfile]: Also fixed the annoying dependency dialogs that popped up.

File: docker/jupyter/env.sh [hunk 2]
```diff
@@ -2,4 +2,6 @@
 
 export JUPYTERHUB_CRYPT_KEY=$(openssl rand -hex 32)
 
+#echo "/jupyter/check-git-config.sh" > $HOME/.bashrc
```
Comment (bjornandre) [inline docker/jupyter/env.sh:5]: Remove this

Comment (oranheim) [reply docker/jupyter/env.sh]: Removed

**Pull request merged**